### PR TITLE
Add vivagraphjs@0.8.2 w/ npm auto-update

### PR DESCRIPTION
**Pull request by mercyg**
Pull request for issue: #10983
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/anvaka/VivaGraphJS
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/vivagraphjs
 * License and its reference: BSD-3-Clause
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:145
   - Count of stars: 2,248
   - Count of forks: 283
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 152
   - Downloads in the last month: 590

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/vivagraphjs/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "keywords": [
+    "graph"
+  ],
+  "author": "Andrei Kashcha",
+  "license": "BSD-3-Clause",
+  "npmName": "vivagraphjs",
+  "npmFileMap": [
+    {
+      "basePath": "",
```
Comment (extend1994) [inline ajax/libs/vivagraphjs/package.json]: We need all files (which corresponds to the rule `**/*`) under `dist` directory, so let's use `dist` as the `basePath` :smiley:

File: ajax/libs/vivagraphjs/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "vivagraphjs",
+  "version": "0.8.2",
+  "description": "Graph Drawing Library",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/anvaka/VivaGraphJS.git"
+  },
+  "keywords": [
+    "graph"
```
Comment (extend1994) [inline ajax/libs/vivagraphjs/package.json]: Please add keywords based on [bower.json](https://github.com/anvaka/VivaGraphJS/blob/master/bower.json).

When the keywords are few in `package.json`, first you can see if there is `bower.json` on npm/GitHub and take it as a reference. Second ( if the keywords are still not good enough ), you can try to find out good keywords by googling them, if the GitHub/npm/homepgae displays on the first page of search result, then it's good!

File: ajax/libs/vivagraphjs/package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "npmName": "vivagraphjs",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "vivagraph.min.js"
```
Comment (clairetsai818) [inline ajax/libs/vivagraphjs/package.json]: We usually put the important info upper in `package.json`, so please move `filename` field prior.

**Comment by mercyg** [comment]
@extend1994 I have updated the PR please review it :)

**Comment by mercyg** [comment]
@extend1994 any update on this?

**Comment by PeterDaveHello** [comment]
Thank you all 😄

**Pull request merged**